enabler/dev-utils/session.js

```diff
@@ -40,7 +40,7 @@ const getSessionId = async(cartId) => {
 
   const sessionMetadata = {
     processorUrl: __VITE_PROCESSOR_URL__,
-    allowedPaymentMethods: ['card'], // add here your allowed methods for development purposes
+    allowedPaymentMethods: ['card', 'klarna_paynow', 'klarna', 'klarna_account'], // add here your allowed methods for development purposes
```

Comment:
Since this is a dev thing only I permanently added the klarna payment methods to this list. Let me know if this is undesired and only want `card` in here. (same applies to the `enabler/index.html`